apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-docs.component.html

```diff
@@ -1,18 +1,20 @@
 <fd-docs-section-title [id]="'def'" [componentName]="'Radio Button Group'">
-    Radio Button Group with Radio buttons as Content
+    Radio Button Group With Radio Buttons Passed as Content.
 </fd-docs-section-title>
 
-<description>This example shows Radio Button Group implementaion with Content as Radio Buttons</description>
+<description>
+    Platform Radio Buttons passed as Content
+</description>
 <component-example>
     <fdp-platform-radio-group-content-example></fdp-platform-radio-group-content-example>
 </component-example>
 <code-example [exampleFiles]="contentRadioGroup"></code-example>
 
 <fd-docs-section-title [id]="'def'" [componentName]="'Disabled Radio Button Group'">
-    Examples for Radio Button Group with List of values
+    Radio Button Group Creates Radio Buttons With List of String Values.
 </fd-docs-section-title>
 <description>
-    This example shows Regular Radio Button Group implementaion with list of string Values.<br />
+    Creating platform radio buttons with list of string Values.<br />
```

Comment:
probably have it as 'Platform Radio Buttons' to be consistent with other lines

Reply:
change Done.